### DISCUSSION TITLE
Warning: "Reward was outside of range"

### DISCUSSION THREAD

**Discussion by team-daniel** [PKU-Alignment/safety-gymnasium#84] [Q&A]
Hi all,

When training an RL agent on one of the envs, I sometimes receive the warning below:

`Warning: reward was outside of range!`

Does anyone know what this alludes to and how to prevent it? Thanks in advance :)

**Comment by muchvo** [comment]
Apologies for missing this discussion earlier. Could you provide more details to help us understand the issue better? Information such as the environment you're running, the reward configuration, and any other relevant settings would be very helpful.

If this message is appearing in the default environment, it typically indicates there might be a bug. Please share the additional details, and we will look into this promptly.

Thank you for bringing this to our attention!
